centralized-sampling-tests/sample-apps/golang-http-server/application.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/contrib/propagators/aws/xray"
+	sampler "go.opentelemetry.io/contrib/samplers/aws/xray"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	"go.opentelemetry.io/otel/sdk/trace"
+	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"google.golang.org/grpc"
+)
+
+func getSampledSpanCount(name string, totalSpans string, attributes []attribute.KeyValue) int {
+	tracer := otel.Tracer(name)
+
+	var sampleCount = 0
+	totalSamples, err := strconv.Atoi(totalSpans)
+	if err != nil {
+		log.Println(err)
+		return -1
```

Comment:
```suggestion
		return 0, err
```
Instead of returning `-1` change your function return values to return the integer and the error. This is more idiomatic.